### PR TITLE
[JBPM-9999] Dispose containers in some jbpm integration tests

### PR DESCRIPTION
**Pull request by gmunozfe**
**JIRA**: [JBPM-9999](https://issues.redhat.com/browse/JBPM-9999)

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Review by afalhambra — APPROVED** [review]
Minor comments.
Nice job @gmunozfe!

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/UserTaskEscalationIntegrationTest.java [hunk 1]
```diff
@@ -123,6 +123,11 @@ public static void buildAndDeployArtifacts() {
         createContainer(CONTAINER_ID, releaseId, PPI_RUNTIME_STRATEGY);
         createContainer(CONTAINER_ID_NOTIFICATION, releaseNotificationId, PPI_RUNTIME_STRATEGY);
     }
+    
+    @AfterClass
+    public static void disposeAndCleanUp() throws Exception {
```
[the remark this answers is not in the document]
Comment (gmunozfe) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/UserTaskEscalationIntegrationTest.java]: done

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/NotificationSaveContentIntegrationTest.java [hunk 2]
```diff
@@ -51,6 +52,11 @@ public static void buildAndDeployArtifacts() {
 
         createContainer(CONTAINER_ID_NOTIFICATION, releaseId);
     }
+    
+    @AfterClass
+    public static void disposeAndCleanUp() throws Exception {
```
[the remark this answers is not in the document]
Comment (gmunozfe) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/NotificationSaveContentIntegrationTest.java]: done

**Comment by gmunozfe** [comment]
@mareknovotny ready to merge

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**